plugin.video.regiotv/addon.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.regiotv" name="Regio TV" version="0.1.0" provider-name="dagwieers">
+<requires>
+  <import addon="script.module.routing" version="0.2.3"/>
+  <import addon="xbmc.python" version="2.25.0" />
```

Comment:
> WARN: For leia it is advised to set xbmc.python version to 2.26.0

Reply:
But we also would like to support Krypton. And not manage 2 versions.